Client/Configuration/Fennec.enterprise.xcconfig

```diff
@@ -13,3 +13,12 @@ INCLUDE_SETTINGS_BUNDLE = YES
 // Adjust SDK Variables
 ADJUST_ENVIRONMENT =
 ADJUST_APP_TOKEN =
+
+// The suffix of the app group identifier used for sharing data across
+// extensions and keychain
+MOZ_GROUP_SUFFIX = org.mozilla.ios.Firefox.enterprise
```

Comment:
Double checking with @sleroux that these `MOZ_GROUP_SUFFIX` changes are correct.